spec/unit/rest/phmap_spec.rb

```diff
@@ -0,0 +1,115 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require 'gooddata/rest/phmap'
+
+PhEx = Struct.new(:name, :title, :mapped)
```

Comment:
Could you please rename `PhEx` to something more descriptive? E.g. `PlaceholderExample`